recipe_scrapers/domesticateme.py

```diff
@@ -4,7 +4,7 @@
 class DomesticateMe(AbstractScraper):
     @classmethod
     def host(cls):
-        return "https://domesticate-me.com/"
+        return "domesticate-me.com/"
```

Comment:
Are we likely to have a similar issue due to trailing slashes in hostnames, I wonder?

Reply:
From having tried to retrieve a recipe, I did not notice this being an issue.  
However, it would indeed be more consistent without slashes, and I can imagine use cases where people would expect to match the domain only.

Reply:
I checked that it's the only host with a slash, so I'll remove it :+1:

Reply:
Thanks!